config/enabled.yml

```diff
@@ -1069,6 +1069,13 @@ Rails/ActionFilter:
   Description: 'Enforces consistent use of action filter methods.'
   Enabled: true
 
+Rails/ActiveRecordAliases:
+  Description: >-
+                  Avoid ActiveRecord aliases:
```

Comment:
Thanks for changing. There is only a little more.

It's better to notate `Active Record` here.

> The proper names of Rails components have a space in between the words, like “Active Support”. ActiveRecord is a Ruby module, whereas Active Record is an ORM.

https://github.com/rails/rails/blob/v5.2.0.rc1/guides/source/api_documentation_guidelines.md#wording

Also CI has failed.
https://travis-ci.org/bbatsov/rubocop/jobs/345560127#L1362

Please run `rake generate_cops_documentation` and commit (and squash) the results.